### DISCUSSION TITLE
Support for module.exports as object property ?

### DISCUSSION THREAD

**Discussion by Arooba-git** [babel/babel#15297] [Q&A]
Hi :) 
I'm parsing official react-documentation website using babel parser, and getting the 'Unexpected token' error here (screenshot below). Does babel parser not support module.exports as an object property or am I missing something? 🤔 

<img width="955" alt="Screen Shot 2022-12-22 at 2 50 03 PM" src="https://user-images.githubusercontent.com/56495631/209066490-55c2ac38-8493-49bf-94b3-f73cc2bbf2f9.png">

Here's my plugin config:
<img width="982" alt="Screen Shot 2022-12-22 at 3 34 53 PM" src="https://user-images.githubusercontent.com/56495631/209072394-7f9dec3e-2771-4fcf-8530-e677aaa27679.png">

**Comment by liuxingbaoyu** [comment]
https://babeljs.io/repl/build/53655/#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&corejs=false&spec=false&loose=false&code_lz=CYUwxgNghgTiAEBbA9sArhBUDeAoe8oksCK6mAdCAB4AOyMALgM4BcAjLgL5A&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=flow&prettier=false&targets=&version=7.20.6%2Bpr.15234&externalPlugins=&assumptions=%7B%7D

This should be parsed, can you try removing the ts plugin temporarily?

**Reply by Arooba-git** [reply]
I had tried without the ts plugin at first :/ should I disable the preset too?

**Reply by Arooba-git** [reply]
Yes, the error is gone if I remove both the ts plugin and the present but now, of course, throws error on all ts syntax including this (interface keyword):
<img width="977" alt="Screen Shot 2022-12-23 at 12 56 49 PM" src="https://user-images.githubusercontent.com/56495631/209268506-a55d2fad-77f1-4bb5-8161-289ed88bfdfe.png">

**Reply by liuxingbaoyu** [reply]
You should judge and enable one of these plugins.

**Reply by Arooba-git** [reply]
I see .. you mean flowtypes are not compatible with typescript preset? I should instead add individual ts plugins?

**Reply by Arooba-git** [reply]
> I see .. you mean flowtypes are not compatible with typescript preset? I should instead add individual ts plugins?

@liuxingbaoyu could you kindly confirm this, please? 😟

**Reply by liuxingbaoyu** [reply]
These two presets should not be enabled at the same time, you should only enable one,
